iterminator/iterminator.py

```diff
@@ -12,6 +12,24 @@
 import getchs
 
 
+LIGHT_SCHEMES = [
```

Comment:
That would definitely be cool. Presumably that would involve inspecting the `*.itermcolors` color scheme file. E.g. `Piatto Light.itermcolors` has this entry, so it does look like this is very doable.

```
        <key>Background Color</key>
        <dict>
                <key>Blue Component</key>
                <real>1</real>
                <key>Green Component</key>
                <real>1</real>
                <key>Red Component</key>
                <real>1</real>
        </dict>
```

Let's create a new issue for doing it properly and allow this one to merge? For this PR I simply hit right arrow lots of times and stopped every time it had a light background.